From: Robert Richter <robert.richter@amd.com>
Date: Fri, 26 Feb 2010 15:01:23 +0100
Subject: [PATCH 02/50] oprofile: remove EXPERIMENTAL from the config option description
Patch-mainline: v2.6.34-rc1
Git-commit: b309a294e5b24692d0f7ea1defa168074cea619e
References: FATE#309728, FATE#309730, BNC#686840
Signed-off-by: Tony Jones <tonyj@suse.de>

OProfile is already used for a long time and no longer experimental.

Signed-off-by: Robert Richter <robert.richter@amd.com>
---
 arch/Kconfig |    2 +-
 init/Kconfig |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/Kconfig b/arch/Kconfig
index 25c1a4a..640555a 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -3,7 +3,7 @@
 #
 
 config OPROFILE
-	tristate "OProfile system profiling (EXPERIMENTAL)"
+	tristate "OProfile system profiling"
 	depends on PROFILING
 	depends on HAVE_OPROFILE
 	select RING_BUFFER
diff --git a/init/Kconfig b/init/Kconfig
index 0078f62..c3b68ca 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1109,7 +1109,7 @@ config DEFAULT_VM_DIRTY_RATIO
 	  Only use this if you really know what you are doing.
 
 config PROFILING
-	bool "Profiling support (EXPERIMENTAL)"
+	bool "Profiling support"
 	help
 	  Say Y here to enable the extended profiling support mechanisms used
 	  by profilers such as OProfile.
-- 
1.7.3.4

